configure.js

```diff
@@ -35,21 +37,12 @@ A configuration has been loaded by default. Make sure to do custom configuration
 };
 
 function defaultConfigure () {
-  if (typeof web3 === 'undefined') {
-    throw new Error(`Cannot find a global Web3 instance. Please configure one manually:
-
-    require('openzeppelin-test-helpers/configure')({ web3: ... });
-
-`
-    );
-  }
-
-  // use global web3
-  setWeb3(web3);
+  setWeb3(new Web3(DEFAULT_PROVIDER_URL));
 }
 
 function customConfigure (config) {
-  setWeb3(config.web3);
+  // The provider may be either an URL to an HTTP endpoint, or a complete provider object
+  setWeb3(new Web3(config.provider));
```

Comment:
Have you considered using `web3.setProvider`, to change the provider of the existing instance instead of creating a new instance? I don't remember if we discused this already.

This would simplify the implementation by removing `getWeb3`/`setWeb3`, and I think it would allow to configure more than once, which has been a pain point for users. (#69)

Reply:
Hm, I guess that could work, but I'm not sure how it'd play out if also using truffle's global web3 instance. Should we in that case have our own web3 instance as well, and set its provider using truffle's?

Reply:
Yes! We should always have our own web3 instance to control the specific version that we're using.